Tests/UnitTests/Fatras/Digitization/UncorrelatedHitSmearerTests.cpp

```diff
@@ -280,4 +295,62 @@ BOOST_AUTO_TEST_CASE(FreeAll) {
   }
 }
 
+BOOST_AUTO_TEST_CASE(GaussianSmearing) {
+  nlohmann::json djson = nlohmann::json::parse(R"(
+    {
+    "acts-geometry-hierarchy-map" : {
+    "format-version" : 0,
+    "value-identifier" : "digitization-configuration"
+    },
+    
+  "entries"
+      : [
+        {
+           "volume" : 1,
+           "value" : {
+            "smearing" : [
+              {"index" : 0, "mean" : 0.0, "stddev" : 0.05, "type" : "Gauss", "forcePositiveValues" : true}
+             
+             
+            ]
+          }
+        }
+      ]
+})");
+  double radius = 5.;
+  double halfZ = 8.;
+  Fixture<ActsExamples::RandomEngine> f(
+      123567,
+      Acts::Surface::makeShared<Acts::StrawSurface>(
+          Acts::Transform3(Acts::Translation3(0., 0., 0.)), radius, halfZ));
+
+  // Get the smearing configuration from the json object
+  auto digiConfig =
+      ActsExamples::DigiConfigConverter("digitization-configuration")
+          .fromJson(djson);
```

Comment:
I wonder if it might be easier to set the digi config directly in the code instead of reading it from json with fake geo ids